### ISSUE TITLE
After Update I can't unfold the “widget-inputs shift” button, and it will throw an error.

### ISSUE THREAD

**Issue by Rayray927** [comfyanonymous/ComfyUI#3291]
![image](https://github.com/comfyanonymous/ComfyUI/assets/91202729/014249e1-6f92-4691-a4c4-8b1c0b46392e)

**Comment by Rayray927** [comment]
![image](https://github.com/comfyanonymous/ComfyUI/assets/91202729/b4681991-fa0f-4454-8811-e1ee8ac34317)

**Comment by ltdrdata** [comment]
This should be reported to rgthree repo.
https://github.com/rgthree/rgthree-comfy/issues

**Comment by rgthree** [comment]
I don't think this one has anything to do with rgthree-comfy (while menu_auto_nest is an rgthree-comfy feature, the error is in a "main.js" which is not from rgthree-comfy).

But also, it looks like maybe ComfyUI fixed it themselves?: https://github.com/comfyanonymous/ComfyUI/commit/a88b0ebc2d2f933c94e42aa689c42e836eedaf3c

(or, at least let you disable the erroneous menu in the settings?)
